apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -496,6 +507,19 @@ const FilterBar = ({
   )
 }
 
+SearchInput.propTypes = {
+  textFilter: PropTypes.string.isRequired,
+  updateTextFilter: PropTypes.func.isRequired,
+}
+
+SearchPopover.propTypes = {
+  visible: PropTypes.bool.isRequired,
+  opener: PropTypes.object,
+  setSearchVisible: PropTypes.func.isRequired,
+  textFilter: PropTypes.string.isRequired,
+  updateTextFilter: PropTypes.func.isRequired,
+}
```

Comment:
I think it probably makes sense to put these `propTypes` declarations right below each of the components they describe.